webhooks/collector.go

```diff
@@ -49,11 +51,12 @@ func SetMaps() {
 	webhookCollections[Weather] = &WebhookList{}
 }
 
-func AddMessage(webhookType string, message interface{}) {
+func AddMessage(webhookType string, message interface{}, areas []string) {
```

Comment:
I would think you would want to pass the areaNames as the expanded structure, not compress into a string only to split this out again

Reply:
yeah thought that too... but did not really find a solution for that. geography.go and stats.go are both in decoder module.

also stats.go relates to pokemon struct etc... maybe you have some time to figure that out? as I told you keeping the struct would lead into a cyclic dependey instead :(

Reply:
Move areaName and the match definition into the geo module
the loader can stay in geography as that is a specific use I guess (or there could be a ReadGeoJson(filename) which returns the structure which geography then becomes much simpler

Reply:
done, thanks for the hint